resources/assets/js/helpers/routes.ts

```diff
@@ -75,3 +75,19 @@ export function managerFaq(locale: string, faqSection?: FaqSection): string {
   }
   return base;
 }
+
+export function hrJobIndex(locale: string): string {
+  return `/${locale}/hr-adviser/jobs`;
```

Comment:
Should do a global find-replace `adviser` > `advisor` just to ensure consistency

Reply:
Ah, you're right, or the other way around. I'll ask about a preferred spelling.